pkg/snapshot/v1alpha1/snapshot.go

```diff
@@ -320,11 +340,15 @@ func (s *snapshot) List() (*v1alpha1.CASSnapshotList, error) {
 	return snapList, nil
 }
 
-func getReadCASTemplate(sc *v1_storage.StorageClass) string {
+func getReadCASTemplate(defaultCasType string, sc *v1_storage.StorageClass) string {
 	castName := sc.Annotations[string(v1alpha1.CASTemplateKeyForSnapshotRead)]
 	// if cas template for the given operation is empty then fetch from environment variables
 	if len(castName) == 0 {
 		casType := strings.ToLower(sc.Annotations[string(v1alpha1.CASTypeKey)])
+		// if casType is missing in sc annotation then use the default cas type
+		if casType == "" {
+			casType = strings.ToLower(defaultCasType)
```

Comment:
Does this work?
Do we pass defaultCasType?
I noticed we passed as bunch of CAS configs as a string type

Reply:
I think `defaultCasType` just a name, its getting extracted from PV object labels and passed to this method.
We can name it something else ?